flutter/lib/src/sentry_flutter.dart

```diff
@@ -155,6 +148,11 @@ mixin SentryFlutter {
     // Will call WidgetsFlutterBinding.ensureInitialized() before all other integrations.
     integrations.add(WidgetsFlutterBindingIntegration());
 
+    options.logger(
```

Comment:
Is this something we always want to print, or only when some condition is met?

Reply:
phew, not sure how that landed in there

this is not part of my PR

Reply:
I'll remove it